apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/transaction/DexOfferTransaction.java

```diff
@@ -129,7 +166,7 @@ public boolean isPhasingSafe() {
 
     @Override
     public String getName() {
-        return "DexOffer";
+        return "DexOrder";
```

Comment:
I supposed it should repeat the table name. Isn't it?
`private static final String TABLE_NAME = "dex_offer";`

Reply:
We don't use db names for transactions